File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> m;
        vector<int> res;
        vector<vector<int>> ans(nums.size()+1);

        for(auto n:nums){
            m[n]++;
        }

        for(auto a:m){
            ans[a.second].push_back(a.first);
        }
        for(auto a=ans.rbegin();a!=ans.rend();++a){
            for(auto b:*a){
                res.push_back(b);
                k--;
                if(k==0)
                    return res;
            }
        }
        return res;
    }
};

File: Data Structures & Algorithms/subsets/submission-1.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res=[]
        def backtrack(nums,s):
            if not len(nums):
                res.append(s)
                return
            s.append(nums[0])
            backtrack(nums[1:],s.copy())
            s.remove(nums[0])
            backtrack(nums[1:],s)

        backtrack(nums,[])
        return res